app/page.tsx

```diff
@@ -1,33 +1,37 @@
 "use client";
-import "./mainpage.css";
 import { Button } from "@mui/material";
 import Link from "next/link";
-import { ThemeProvider } from "@mui/material/styles";
-import theme from "./theme";
+import ReactVirtualizedTable from "@/components/GameTable";
 
 export default function MainPage() {
+	async function showGames() {
+		const response = await fetch("api/games");
+		const games = await response.json();
+		return games;
+	}
+	showGames();
```

Comment:
Let's leave listing page for another PR, these changes can be removed from this PR as they're not tackling anything related to te "create game" page.

PS. you may want to read through next documentation about data fetching [here](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating)